### ISSUE TITLE
Annoying log messages about fail to establish a failover path every 60s

### ISSUE THREAD

**Issue by italovalcy** [kytos-ng/mef_eline#488]
Hi,

The Mef_Eline consistency check routine tries to establish a failover path for the EVCs on each execution. This is fine for most cases, where you want to have a failover path whenever available. For scenarios where the failover_path is not possible, the log messages generated are quite annoying (in the sense that there is nothing you can do about it, so there is no advantage of that particular logging).

How to reproduce? 1) create a linear topology and create a dynamic EVC. You will see that after the EVC gets enabled, every 60s you will log messages like:

```
kytos.napps.kytos/mef_eline:WARNING evc:973:  Failover path for EVC(abc73a657c2140, my evc1) was not deployed: No available path was found.
```

One suggestion is to add an additional parameter to the setup_failover_path to avoid logging on this particular call. 

To be clear: the above log message is pretty useful on all other scenarios, but not on this particular case. That is why having a parameter to ignore logging on this scenario would fit well, IMHO.

**Comment by viniarck** [comment]
Right. Yes, conditionally parametrizing to log or not the warning is a good compromise based on how everything else has been structured. Maybe something like `warn_if_not_path=False`.

**Comment by viniarck** [comment]
Probably a good candidate for `2024.2`, there are adjacent changes being prioritized too. Let's see.
